pkg/types/interfacetests/chain_components_interface_tests.go

```diff
@@ -16,6 +16,40 @@ import (
 	"github.com/smartcontractkit/chainlink-common/pkg/values"
 )
 
+// GetLatestValue method
+const GetLatestValueAsValuesDotValue = "Gets the latest value as a values.Value"
+const GetLatestValueNoArgumentsAndPrimitiveReturnAsValuesDotValue = "Get latest value without arguments and with primitive return as a values.Value"
+const GetLatestValueNoArgumentsAndSliceReturnAsValueDotValue = "Get latest value without arguments and with slice return as a values.Value"
+const GetLatestValue = "Gets the latest value"
+const GetLatestValueWithPrimitiveReturn = "Get latest value without arguments and with primitive return"
+const GetLatestValueBasedOnConfidenceLevel = "Get latest value based on confidence level"
+const GetLatestValueFromMultipleContractsNamesSameFunction = "Get latest value allows multiple contract names to have the same function "
+const GetLatestValueWithModifiersUsingOwnMapstrctureOverrides = "Get latest value wraps config with modifiers using its own mapstructure overrides"
+const GetLatestValueNoArgumentsAndSliceReturn = "Get latest value without arguments and with slice return"
+
+// GetLatestValue event
+const GetLatestValueGetsLatestForEvent = "Get latest value gets latest event"
+const GetLatestValueBasedOnConfidenceLevelForEvent = "Get latest event based on provided confidence level"
+const GetLatestValueReturnsNotFoundWhenNotTriggeredForEvent = "Get latest value returns not found if event was never triggered"
+const GetLatestValueWithFilteringForEvent = "Get latest value gets latest event with filtering"
+
+// BatchGet
+const BatchGetLatestValue = "BatchGetLatestValues works"
+const BatchGetLatestValueNoArgumentsPrimitiveReturn = "BatchGetLatestValues works without arguments and with primitive return"
+const BatchGetLatestValueMultipleContractNamesSameFunction = "BatchGetLatestValues allows multiple contract names to have the same function Name"
+const BatchGetLatestValueNoArgumentsWithSliceReturn = "BatchGetLatestValue without arguments and with slice return"
+const BatchGetLatestValueWithModifiersOwnMapstructureOverride = "BatchGetLatestValues wraps config with modifiers using its own mapstructure overrides"
+const BatchGetLatestValueDifferentParamsResultsRetainOrder = "BatchGetLatestValues supports same read with different params and results retain order from request"
+const BatchGetLatestValueDifferentParamsResultsRetainOrderMultipleContracts = "BatchGetLatestValues supports same read with different params and results retain order from request even with multiple contracts"
+const BatchGetLatestValueSetsErrorsProperly = "BatchGetLatestValues sets errors properly"
```

Comment:
Grouping would reduce some noise and provide something to anchor these comments:
```suggestion
// BatchGet
const (
	BatchGetLatestValue = "BatchGetLatestValues works"
	BatchGetLatestValueNoArgumentsPrimitiveReturn = "BatchGetLatestValues works without arguments and with primitive return"
	BatchGetLatestValueMultipleContractNamesSameFunction = "BatchGetLatestValues allows multiple contract names to have the same function Name"
	BatchGetLatestValueNoArgumentsWithSliceReturn = "BatchGetLatestValue without arguments and with slice return"
	BatchGetLatestValueWithModifiersOwnMapstructureOverride = "BatchGetLatestValues wraps config with modifiers using its own mapstructure overrides"
	BatchGetLatestValueDifferentParamsResultsRetainOrder = "BatchGetLatestValues supports same read with different params and results retain order from request"
	BatchGetLatestValueDifferentParamsResultsRetainOrderMultipleContracts = "BatchGetLatestValues supports same read with different params and results retain order from request even with multiple contracts"
	BatchGetLatestValueSetsErrorsProperly = "BatchGetLatestValues sets errors properly"
)
```

Reply:
Done